go.mod

```diff
@@ -258,6 +256,11 @@ require (
 	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
 )
 
+require (
+	github.com/Azure/go-autorest/autorest/adal v0.9.15 // indirect
```

Comment:
No idea why these got moved into its own section, but if go mod tidy doesn't fix it - fine